scripts/configure

```diff
@@ -216,9 +216,11 @@ function gn_generate() { # [project options]
     } >"${BUILD_DIR}/args.gn"
     "${gn[@]}" -q gen "$BUILD_DIR"
 
+    # Generate args.configured first
```

Comment:
This comment does not seem to explain the code but rather re-read and seems to be PR-related. On the whole PR, this is not actually "first" as for example the "gn -q gen" above is run before it.

This seems to need some different text, like why we are doing this before args.gn. This should state `Generate args.configured before args.gn because ...`

Reply:
You are totally right. I improved the comment. Thanks for the feedback!